app/src/main/java/com/moxielabs/androidarchitectureexamples/mvc/MvcFragment.kt

```diff
@@ -18,7 +18,7 @@ class MvcFragment : Fragment() {
         inflater: LayoutInflater, container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View? {
-        return inflater.inflate(R.layout.fragment_mvc, container, false)
+        return inflater.inflate(R.layout.fragment_mvc_mvp, container, false)
```

Comment:
Wondering if we should just reuse the same layout since it's literally all using the same item_layout 🤔

Reply:
Yep. I figured that we would eventually end up there. this is for the fragment layout. were you referring to the list item layout instead?

Reply:
I was thinking more of just the name - naming it something like `fragment_architecture` or something since each of the architectures can use the same fragment layout. I think that would be cleaner than adding on each arch to the name